### ISSUE TITLE
How to improve cleaning in the Office

### ISSUE THREAD

**Issue by xaviperalta** [Casumo/BarcelonaOffice#5]

**Comment by charlotteros** [comment]
Hi Xavi, thanks a lot for this!

To be honest, I think that the guidelines don't have to be much of a brain buster. Basically what we should tell her is to do everything properly. Because what happens now is that she cleans the visible surfaces, but then that's it. 

For example: 
-She mops the floor, but doesn't pull the trashcans out of the way. 
-She wipes the table, but doesn't remove whatever is on the table.
-She wipes the toilet seat, but doesn't clean the actual toilet bowl.
-She rinses the glasses, but doesn't clean the sink.

What Andy and I noticed yesterday was the stuff mentioned above plus that she didn't touch the shelves with all the food and coffee, which was still covered in crumbs (thinking about the cockroaches here) and lots of coffee and tea staines.

I think Emilia is really sweet, but she just needs to be more thorough, that's all. I think just tell her that and give her a chance to improve.

**Comment by raquelvila** [comment]
@xaviperalta have they been provided with a checklist of what's expected from them/her?
I might sound too much or unnecessary but maybe it could help (really granular checklist)
On the other hand I would talk with the person in charge to make them understand we are not happy with the standards and that we'll try from our end to communicate better what we expect from them but if it doesn't work out we'll have to find another options
(my 2cents)

**Comment by CaroPodeswa** [comment]
Hey there,
I want to add that Emilia(?) does an absolutely amazing job in the Llull apartment. When I forget to tidy up papers and stuff on my table or drawer in the morning(which happens, because I'm a sleepy, unthoughtful idiot sometimes), she organizes them and puts it to the side, so that she can clean the surface. 

On another note, it seems like the fact that we have a cleaning lady makes some people a bit... let's call it 'careless'? There are often crumbs and food leftovers on the tables of the 4th floor, it's no wonder there are cockroaches in the building, but I would not blame the cleaning lady for that.

Or the one time when there were lots of beer bottles and deliveroo food bags on the 3rd floor, after there was a gaming night. We had cleaning ladies in my former companies too, but it was never a problem or even topic to clean after the mess that you make yourself. That's just common sense and just being adult about it.

To sum it up: She keeps the company apartment clean, and we all(Mario, Nakul&me) like her. I hope we can just tell her about the things she might have been sloppy with at the office.

**Comment by raquelvila** [comment]
Thanks for your comment caro, maybe it's just a matter of communication then
